ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/BridgeConfig.scala

```diff
@@ -89,15 +92,23 @@ object BridgeConfigProvider extends ConfigProvider[BridgeConfig] {
       case TimeProviderType.WallClock => None
     }
 
+  override def initialLedgerConfig(config: Config[BridgeConfig]): InitialLedgerConfiguration = {
+    val superConfig = super.initialLedgerConfig(config)
+    superConfig.copy(configuration =
+      superConfig.configuration.copy(maxDeduplicationTime = DefaultMaximumDeduplicationTime)
+    )
+  }
+
   override val defaultExtraConfig: BridgeConfig = BridgeConfig(
     // TODO SoX: Enabled by default
     conflictCheckingEnabled = false,
-    maxDedupSeconds = 30,
     submissionBufferSize = 500,
     implicitPartyAllocation = false,
     timeProviderType = TimeProviderType.WallClock,
     authService = AuthServiceWildcard,
     profileDir = None,
     stackTraces = false,
   )
+
+  val DefaultMaximumDeduplicationTime: Duration = Duration.ofHours(1L)
```

Comment:
This is 5.4M de-duplication records when transaction rate is at 1.5k. Is that a reasonable number?

Reply:
I guess it doesn't matter all that much since you can override it...

Reply:
This is a linked map of change ids and timestamps. It should be well below 1GB. And indeed it is overridable

Reply:
But why have it by default so high, you're right. Changed it to 5 minutes. The allowed max is still 1 hour